admin_auto_filters/filters.py

```diff
@@ -72,11 +76,16 @@ def __init__(self, request, params, model, model_admin):
         )
 
     def get_queryset_for_field(self, model, name):
-        field_desc = getattr(model, name)
+        try:
+            field_desc = getattr(model, name)
+        except:
```

Comment:
```suggestion
        except AttributeError:
```

Reply:
Yup, good call. In general, I did want to make sure that the changes I made here made sense. I tend to have a fair number of ForeignKey relations that don't have a related_name or related_query_name. These changes worked for my cases, but perhaps there should be testing to cover some additional scenarios.

Reply:
Ah!  In that case more exceptions would be needed in the clause, or `except Exception` to simplify, but I follow the rule to avoid bare excepts in 99% of cases.

Reply:
Completely agree on the `AttributeError`. I added that, as well as some tests to try to ensure that this method works for all cases.